client/state/experiments/selectors.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * External Dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { AppState } from 'types';
+
+/**
+ * Returns the user's assigned variation for a given experiment
+ *
+ * @param state The application state
+ * @param experiment The name of the experiment
+ */
+export const getVariationForUser = ( state: AppState, experiment: string ) =>
+	get( state, [ 'experiments', 'variations', experiment ], null );
+
+/**
+ * Returns true if the variations are loading for the current user
+ *
+ * @param state The application state
+ */
+export const isLoading = ( state: AppState ) => get( state, [ 'experiments', 'isLoading' ], true );
```

Comment:
If `state.experiments.isLoading` doesn't exist or is undefined for some reason, shouldn't the default value be `false`?

Reply:
That's a good point. I'll update it.